src/index.js

```diff
@@ -1,47 +1,174 @@
 import './style.css';
-
 // input section
 import Image from './asset/images/add-grey.svg';
 import Image1 from './asset/images/3dots-grey.svg';
+import Image2 from './asset/images/trash.svg';
 
 const render = `<a href="#"><img src="${Image}" alt="img"></a>`;
 const addButton = document.getElementById('add-button');
 addButton.innerHTML = render;
 
 // list section
-const ListContent = [
-  {
-    id: 0,
-    description: 'task1',
-    status: false,
-  },
-  {
-    id: 1,
-    description: 'task2',
-    status: false,
-  },
-  {
-    id: 2,
-    description: 'task3',
-    status: false,
-  },
-  {
-    id: 3,
-    description: 'task4',
-    status: false,
-  },
-];
-
-// iteration through array
-ListContent.forEach((todo) => {
+let ListContent = [];
+
+// functions
+// status checkbox
+
+const statusCheckBox = (e) => {
+  const list = e.target.parentNode.parentNode;
+  list.children[0].children[1].classList.toggle('strike');
+  const completed = !list.classList.contains('disabled');
+  list.classList.toggle('disabled');
+  const getLists = document.querySelector('#ulList').children;
+  const index = Array.from(getLists).indexOf(list);
+  ListContent[index].completed = completed;
+  localStorage.setItem('tasks', JSON.stringify(ListContent));
+};
```

Comment:
- [ ] Right now, I can only delete completed todos only by clicking the clear all button. Todos that are marked as complete can't be deleted or edited(the options icon is disabled), Users should be able to delete/edit completed todos.

Reply:
Thanks, @omar-labana, for reviewing my project. That was nice you experimented with all the options with my project. I totally appreciate it. 
Let me share my thoughts here. I made that work that way, so that if user inputs are marked as checked it's locked and can't be edited accidentally. When a user wants to change something in the todo-list tasks, the user needs to mark uncompleted, and then he can edit it. That's why i made it work like that. 
But I can change it and make it work as you mentioned :smile:  
Thank you :pray: